spec/rspec/que/queue_up_spec.rb

```diff
@@ -24,6 +24,17 @@ class CJob; end
       expect(instance.failure_message).
         to eq("expected to enqueue a job, but found nothing")
     end
+
+    context "when it enqueues a job and we allow anything" do
+      let(:job_class) { anything }
+
+      it { is_expected.to be(false) }
+      specify do
+        matches?
+        expect(instance.failure_message).
+          to eq("expected to enqueue a job any class, but found nothing")
```

Comment:
Does this read better? 🤔

```suggestion
          to eq("expected to enqueue a job of any class, but found nothing")
```

Reply:
Yeah sounds good